Modules/Book HTML.js

```diff
@@ -0,0 +1,9 @@
+export default function appendBookHTML(title, author, element) {
+  const newBook = document.createElement('div');
+  newBook.setAttribute('class', 'book-list');
+  const newEntry = document.createElement('div');
+  newEntry.innerHTML = `<span> "${title}" by ${author} </span><button class="remove">Remove</button>`;
+  newBook.appendChild(newEntry);
+  element.appendChild(newBook);
+  return false;
+}
```

Comment:
- [ ] According to the requirements, you should refactor your methods and functions to arrow functions, so Kindly update it 👍

![image](https://user-images.githubusercontent.com/45231276/205899167-8a030833-29f1-4c1f-88ea-38789c19f6f3.png)